Assets/Scripts/Core/GameManager.cs

```diff
@@ -143,10 +153,49 @@ public float GetFindingFloatDistance()
         return configurator.findingFloatDist;
     }
 
-    public float GetFindingsCount()
+    public float GetExtraBombFindingsCount()
     {
-        // TODO This method's result should be dependent on current level number value!
-        return configurator.initialFindingsCount;
+        if(extraBombFindingsCount < 0)
+        {
+            extraBombFindingsCount = configurator.initialExtraBombFindings;
+        }
+        return extraBombFindingsCount;
+    }
+
+    public float GetExtraLiveFindingsCount()
+    {
```

Comment:
Warto byłoby to zrobić tak aby manager strategii mógł w łatwy sposób modyfikować wartości dla nowych poziomów, a konfiguracja jest tylko dla startowego poziomu